pyproject.toml

```diff
@@ -1,8 +1,65 @@
 [build-system]
-# setuptools and setuptools_scm version are explicitly set
-# to allow support for python2
-requires = ["setuptools>=44", "wheel", "setuptools_scm>=5"]
+requires = ["setuptools>=69", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "deepmerge"
+description = "A toolset for deeply merging Python dictionaries."
+authors =  [
+    {name = "Yusuke Tsutsumi"},
```

Comment:
@toumorokoshi - If you want to include an email address here let me know.

Reply:
go for it! its yusuke@tsutsumi.io